core/internal/cltest/cltest.go

```diff
@@ -95,8 +95,11 @@ const (
 	APIKey = "2d25e62eaf9143e993acaf48691564b2"
 	// APISecret of the fixture API user.
 	APISecret = "1eCP/w0llVkchejFaoBpfIGaLRxZK54lTXBCT22YLW+pdzE4Fafy/XO5LoJ2uwHi"
-	// APIEmail is the email of the fixture API user
-	APIEmail = "apiuser@chainlink.test"
+	// Collection of test fixture DB user emails per role
+	APIEmailAdmin       = "apiuser@chainlink.test"
+	APIEmailEdit        = "apiuser-edit@chainlink.test"
+	APIEmailEditMinimal = "apiuser-edit-minimal@chainlink.test"
+	APIEmailViewOnly    = "apiuser-view-only@chainlink.test"
```

Comment:
Does `Only`/`-only` add value here?
```suggestion
	APIEmailRun = "apiuser-edit-minimal@chainlink.test"
	APIEmailView = "apiuser-view@chainlink.test"
```

Reply:
Personally `ViewOnly` /  `apiuser-view-only@chainlink.test` is more clear and explicit to me than just view 🙂